internal/lib/lib.go

```diff
@@ -184,6 +184,31 @@ func GetSniPGName(ingName, namespace, host, path string) string {
 	return NamePrefix + namespace + "-" + host + path + "-" + ingName
 }
 
+// evh child
+func GetEvhVsPoolNPgName(ingName, namespace, host, path string, args ...string) string {
+	path = strings.ReplaceAll(path, "/", "_")
+	poolName := NamePrefix + namespace + "-" + host + path + "-" + ingName
+	if len(args) > 0 {
+		svcName := args[0]
+		poolName = poolName + "-" + svcName
+	}
+	return poolName
+}
+
+func GetEvhTlsNodeName(ingName, namespace, secret string, host string, path string) string {
```

Comment:
Does the EVH model require separate name for TLS Nodes? I thought in EVH both secure/insecure VSes look the same?

Reply:
Yes the name of the VS, Pool and PG are the same ( having both host and path details. ) Had retained this for the secret name appended to EVH child name, will remove this.